README.md

```diff
@@ -1,25 +1,40 @@
-# GDISF Teaching Materials
-
-Teaching materials for HTML, CSS, JS, and HTML5. We maintain our Open Source materials on this repo. 
-
-## Contributing
-
-[Read our docs](CONTRIBUTING.md)
-
-## Curriculum Creation
-* Read [So You Want To Teach A Workshop](../../wiki/Teaching-a-Workshop)
-* Read our [Curriculum Creation](../../wiki/Curriculum-Creation) Guide in the wiki
-* Make sure that all curricula are in line with the GDI mission and an appropriate topic/level for our members. This may include:
-	* Topics that complement and expand our Front End Web Development curriculum
-	* Industry-relevant topics for advanced students (these students already work in the industry and range from Associate/Junior to Senior and above)
-	* Topics that students have requested to learn
-* Would you like to fork an existing workshop? Check out these resources:
-	[GDI Curriculum Review site](http://girldevelopit.github.io/gdi-curriculum-site/)
-	[girldevelopit.com](https://www.girldevelopit.com/materials)
-* Make a [Pull Request](https://github.com/gdisf/teaching-materials). You may want to use the `example/` directory to get started with the required templates. Your PR must must include:
-	- [ ] Description
-	- [ ] Lecture slides
-	- [ ] Hands-on exercises
-	- [ ] Solutions to exercises
-	- [ ] Follow Up Email
-* If you want your workshop to be included in our [national website](https://www.girldevelopit.com/materials), check out this [review rubric](https://docs.google.com/document/d/1zw7qX2EAO08rGAjl9683aVxtsAoZQLxmy7evu2my4OY/edit). Please note that this is not a requirement for inclusion in this repository.
+# GDI Seattle Algorithms 101 - Forked from GDISF Teaching Materials
```

Comment:
Update this to `GDI Seattle Computer Science Fundamentals - Forked from GDISF Teaching Materials`